### PR TITLE
Reworking make/produce scripts to use autoupload

### PR DESCRIPTION
**Pull request by AlisonLanski**

### CODE REVIEW

**Comment by AlisonLanski** [comment]
Resolves #71 #82 #

File: R/write_report.R [hunk 1]
```diff
@@ -1,85 +1,39 @@
...
 
-write_report <- function(df, component, part, output, append = FALSE, format = "uploadable") {
+write_report <- function(..., survey, part, output_path) {
+
+  fullpath <- paste0(output_path, survey, "_", part, "_", Sys.Date(), ".txt")
 
-  while (!exists(x = "output_path", envir = globalenv())) {
-    set_report_path()
-  }
+  dfs <- list(...)
```
Comment (shilohfling) [inline R/write_report.R:24]: WHOAAA

File: R/make_com_part_D.R [hunk 2]
```diff
@@ -2,20 +2,19 @@
...
 #' @importFrom stringr str_to_upper
 #'
 #' @return A text file
 #' @export
 #'
 
-make_com_part_D <- function(df, extracips = NULL, output = "part", format = "both") {
+make_com_part_D <- function(df, extracips = NULL) {
+
+  stopifnot()
```
Comment (shilohfling) [inline R/make_com_part_D.R:17]: What do we want to stop?

**Review by shilohfling — APPROVED** [review]

File: R/make_com_part_D.R (hunk 2 above)
Comment (AlisonLanski) [reply R/make_com_part_D.R]: whoops, nothing...

Comment (AlisonLanski) [reply R/make_com_part_D.R]: whoops, nothing...

**Pull request merged**